metaseq/model_parallel/criterions/vocab_parallel_cross_entropy.py

```diff
@@ -46,7 +46,11 @@ def forward(self, model, sample, reduce=True):
         if has_pad:
             loss = loss * (target != self.padding_idx)
         loss = loss.sum()
-        sample_size = sample["ntokens"]
+        sample_size = (
+            sample["ntokens_target"]
```

Comment:
Nit: add a comment for what this logic is for / where `ntokens_target` is present vs `ntokens`?

Reply:
Will add